.github/workflows/ci-workflow.yml

```diff
@@ -85,22 +81,27 @@ jobs:
       - name: Add MSBuild to PATH
         uses: microsoft/setup-msbuild@v1.0.0
 
-      - name: Install chocolatey packages
-        run: choco install ninja nsis
+      - name: Install chocolatey packages (i386)
+        if: ${{ matrix.cfg.arch == 'Win32' }}
+        run: choco install ninja nsis openssl -y --x86
+
+      - name: Install chocolatey packages (x64)
+        if: ${{ matrix.cfg.arch == 'x64' }}
+        run: choco install ninja nsis openssl -y
 
       - name: Generate CMake
-        run: mkdir main/build && cd main/build && cmake -G "Visual Studio 16 2019" -A "${{matrix.arch}}" ..
+        run: mkdir main/build && cd main/build && cmake -G "Visual Studio 16 2019" -A "${{matrix.cfg.arch}}" -DOPENSSL_ROOT_DIR="${{matrix.cfg.ssl-dir}}" ..
```

Comment:
Is OPENSSL_ROOT_DIR needed again? Or is this just needed in CI? If it's needed again, we should update the build instructions right?

Reply:
It’s just for CI, since CI has multiple 64Bit OpenSSL versions. CMake uses OPENSSL_ROOT_DIR as a hint to know where to look for it. If you one have one OpenSSL that is in the “cmake OpenSSL search path” you don’t need it. But we have two, so it’s used to distinguish.